src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const prop of actions) {
+    switch (prop.type) {
+      case 'addProperties':
+        for (const extra in prop.extraData) {
+          state[extra] = prop.extraData[extra];
+        }
```

Comment:
```suggestion
        Object.assign(state, prop.extraData);
```
I'm not sure but maybe it will be better than loop

Reply:
Yeah, i agree with u, thanks